notebook/services/kernels/kernelmanager.py

```diff
@@ -52,6 +58,26 @@ def _update_root_dir(self, proposal):
             raise TraitError("kernel root dir %r is not a directory" % value)
         return value
 
+    cull_kernels_after_minutes_env = 'CULL_KERNELS_AFTER_MINUTES'
```

Comment:
Providing a way to set traitlets with environment variables is done in the kernel gateway code base, but it's not common in the notebook codebase.